### ISSUE TITLE
Cannot create class with multiple inheritance.

### ISSUE THREAD

**Issue by maggiolo00** [orientechnologies/orientdb#4060]
I got this error but the class is created without superclasses.

{"errors":[{"code":500,"reason":500,"content":"com.orientechnologies.orient.core.exception.OSchemaException: Can't add base class 'Test45', because of recursion"}]}

To Replicate

```
alter class V superclasses ORestricted
create class Dummy1 extends V,ORestricted
create class Dummy2
create class Dummy3 extends Dummy1, Dummy2
```

The class Dummy3 is created with no superclasses

**Comment by PhantomYdn** [comment]
Thank you for the testcase! Fixed in PR #4062 . As a workaround for 2.1-rc1 you can use the following:

```sql
alter class V superclasses ORestricted
create class Dummy1 extends V
create class Dummy2
create class Dummy3 extends Dummy1, Dummy2
```

**Comment by tglman** [comment]
merged the PR closing
